dozer-ingestion/src/connectors/delta_lake/connector.rs

```diff
@@ -0,0 +1,60 @@
+use crate::connectors::delta_lake::reader::DeltaLakeReader;
+use crate::connectors::delta_lake::schema_helper::SchemaHelper;
+use crate::connectors::delta_lake::ConnectorResult;
+use crate::connectors::{Connector, TableInfo, ValidationResults};
+use crate::ingestion::Ingestor;
+use dozer_types::ingestion_types::DeltaLakeConfig;
+use dozer_types::types::SourceSchema;
+use std::collections::HashMap;
+
+#[derive(Debug)]
+pub struct DeltaLakeConnector {
+    pub id: u64,
+    config: DeltaLakeConfig,
+}
+
+impl DeltaLakeConnector {
+    pub fn new(id: u64, config: DeltaLakeConfig) -> Self {
+        Self { id, config }
+    }
+}
+
+impl Connector for DeltaLakeConnector {
+    fn validate(&self, _tables: Option<Vec<TableInfo>>) -> ConnectorResult<()> {
+        Ok(())
```

Comment:
Can we have at least basic validation, which checks that folder exist

Reply:
Seems deltalake crate only has one way to check table path if exists: `deltalake::open_table(path)`, but it's async.

So I'll check if table in config instead.